src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -219,6 +320,38 @@ public function prepare_item_for_response( $theme, $request ) {
 	}
 
 	/**
+	 * Prepares links for the request.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param WP_Theme $theme Theme data.
+	 * @return array Links for the given block type.
+	 */
+	protected function prepare_links( $theme ) {
+		return array(
+			'collection' => array(
+				'href' => rest_url( sprintf( '%s/%s', $this->namespace, $this->rest_base ) ),
+			),
+			'self'       => array(
+				'href' => rest_url( sprintf( '%s/%s/%s', $this->namespace, $this->rest_base, $theme->get_template() ) ),
+			),
+		);
+	}
+
+	/**
+	 * Helper function to compare theme to current theme.
+	 *
+	 * @param WP_Theme $theme  Theme to compare to active theme.
+	 * @param WP_Theme $current_theme Current active theme.
+	 *
+	 * @return bool
+	 */
+	protected function is_current_theme( $theme, $current_theme ) {
```

Comment:
It seems like this should either be called `is_same_theme` or remove the `$current_theme` parameter and call `wp_get_theme` in the method.